src/router.ts

```diff
@@ -129,11 +129,12 @@ class Router {
 
       const clientOrderId = `clientOrderId${Date.now()}`;
       const side = 'SELL';
-      const quantity = dipDeposit.qtyTokens;
+      const quantityDIP = dipDeposit.qtyTokens;
+      const quantityTrade = Math.min(quantityDIP, remainingQuantity);
```

Comment:
do not send a request if remainingQuantity is zero. The getPrice isnt fully supported, so if it breaks, we shouldnt break the public facing endpoints

Reply:
Better?

Reply:
yes